Books-Labels/schema.sql

```diff
@@ -0,0 +1,24 @@
+CREATE DATABASE catalog;
+
+CREATE TABLE item (
+    id  int PRIMARY KEY,
+    publish_date DATE,
+    archived boolean,
+    FOREIGN KEY (label_id) REFERENCES labels (id)
+);
+
+CREATE TABLE books (
+    id integer PRIMARY KEY,
+    title varchar(100),
+    publisher varchar(100),
+    cover_state varchar(100),
+    publish_date,
+    archived boolean,
+    FOREIGN KEY(id) REFERENCES item(id)
```

Comment:
You can't make a primary key field to be a foreign key. Please add a column called `item_id` and make it a foreign key that references `item(id)`.